components/security-mgt/org.wso2.carbon.security.mgt.ui/src/main/resources/web/keystoremgt/import-cert.jsp

```diff
@@ -74,7 +76,7 @@
 
 
     <div id="middle">
-        <h2><fmt:message key="import.certificates.to"/><%=" "+keyStore %></h2>
+        <h2><fmt:message key="import.certificates.to"/><%= " " + Encode.forHtmlContent(keyStore) %></h2>
```

Comment:
Better to use Encode.forHtml here since within <h2> tags other html tags can be used